test/Analyzers.Tests/Orchestration/DateTimeOrchestrationAnalyzerTests.cs

```diff
@@ -45,7 +45,6 @@ public async Task NonOrchestrationHasNoDiag()
     [Theory]
     [InlineData("DateTime.Now")]
     [InlineData("DateTime.UtcNow")]
-    [InlineData("DateTime.Today")]
```

Comment:
Why did you remove this line?

Reply:
I extract that use case into a brand new `Fact` unit test. I did it because the expected fix for `DateTime.Today` is different from `Now` and `UtcNow`. So it was more a refactor than removing a test.